docs/managed-datahub/managed-datahub-overview.md

```diff
@@ -1,8 +1,134 @@
-# Managed DataHub Exclusives
+# How Acryl DataHub compares to DataHub
 
-Acryl DataHub offers a slew of additional features on top of the normal OSS project.
+DataHub is the #1 open source metadata platform for developers.
+Acryl DataHub takes DataHub to the next level by offering features that allow
+you to roll out the product to the entire organization beyond your central data
+platform team.
 
-## Chrome Extension
+We’ve organized this comparison using a few categories:
+* Search and Discovery: the ability to discover data assets at your organization and understand relationships between them
+* Data Governance: features that help you govern the crown jewels of your organization, and trim out the datasets that seem to grow like weeds when no one’s looking
+* Data Quality: features that help you ensure your data pipelines are producing high quality assets, and if they’re not, you’re the first to know
+* Enterprise Grade: features needed to roll out at scale to large enterprises
+* Implementation and Support: features related to ease of deployment and maintenance
+
+## How to read this list
+
+Not a fan of checkboxes? Neither are we. 
```

Comment:
Totally agree that checkboxes would be bland here, but the current styling is pretty tough to read. At a glance, they look like previously-visited hyperlinks; it's also really hard to see in dark mode: 
![CleanShot 2024-04-16 at 17 53 43](https://github.com/datahub-project/datahub/assets/15873986/6f70934e-e6e4-4bea-9397-13d9ac7fc30d)

Reply:
Also, while scrolling down, it's hard to keep the mental model of which color maps to which - by the time I got here, I couldn't easily synthesize which was DataHub vs. Acryl DataHub: 
![CleanShot 2024-04-16 at 17 53 54](https://github.com/datahub-project/datahub/assets/15873986/1e211d09-7fa9-489c-97c0-dd9c6028ca1a)

Reply:
Agree with this - I would rather make this into markdown table with checkboxes, something like this ; https://datahubproject.io/docs/api/datahub-apis#datahub-api-comparison

Reply:
Also, i think we wouldnt need "How to read this list"  if we go with the checkboxes. @shirshanka let me know if you need help with this.